common.sh

```diff
@@ -3,7 +3,9 @@
 CLOUDCUTTER_AP_PREAMBLE="A-"
 TUYA_AP_PREAMBLE="SmartLife-"
 IHOME_AP_PREAMBLE="iHome-"
-COMBINED_AP_PREAMBLE="(${CLOUDCUTTER_AP_PREAMBLE})|(${TUYA_AP_PREAMBLE})|(${IHOME_AP_PREAMBLE})"
+MERKURY_AP_PREAMBLE="Merkury-"
+GEENI_AP_PREAMBLE="Geeni-"
+COMBINED_AP_PREAMBLE="(${CLOUDCUTTER_AP_PREAMBLE})|(${TUYA_AP_PREAMBLE})|(${IHOME_AP_PREAMBLE})|(${MERKURY_AP_PREAMBLE})|(${GEENI_AP_PREAMBLE})"
```

Comment:
This merged regex is unwieldy now. What about having the preambles in a bash array and merging them programmatically?